tuneme/templatetags/surveypolls.py

```diff
@@ -11,3 +11,8 @@ def get_surveytype():
 @register.assignment_tag
 def get_pollstype():
     return "polls"
+
+
+@register.assignment_tag
+def get_competitiontype():
```

Comment:
perhaps better to say get_survey_type

Reply:
we already have 

@register.assignment_tag
def get_surveytype():
    return "surveys"

in this file

Reply:
all of these should be in one template tag, with some logic to return the correct type, we can do it on a call